frontend/fingertips-frontend/components/organisms/LineChart/index.tsx

```diff
@@ -1,44 +1,50 @@
-import { WeatherForecast } from '@/generated-sources/api-client';
+'use client';
+
 import Highcharts from 'highcharts';
 import { HighchartsReact } from 'highcharts-react-official';
+import {
+  generateSeriesData,
+  sortHealthDataByDate,
+} from '@/lib/chartHelpers/formatLineChartValues';
+import { HealthDataForArea } from '@/generated-sources/ft-api-client';
 
 interface LineChartProps {
-  data: WeatherForecast[];
-  title?: string;
+  data: HealthDataForArea[];
   xAxisTitle?: string;
-  yAxisTitle?: string;
   accessibilityLabel?: string;
 }
-
 export function LineChart({
   data,
-  title,
   xAxisTitle,
-  yAxisTitle,
   accessibilityLabel,
 }: Readonly<LineChartProps>) {
-  const categories = data.map(
-    (item) => item.date?.toLocaleDateString('en-GB') ?? ''
-  );
-  const temperatureData = data.map((item) => item.temperatureC ?? '');
+  const sortedSeriesValues = sortHealthDataByDate(data);
+  const seriesData = generateSeriesData(sortedSeriesValues);
 
   const lineChartOptions: Highcharts.Options = {
```

Comment:
I think if you add the following to the option object it'll take the "highcharts.com" watermark off the chart:
```suggestion
  const lineChartOptions: Highcharts.Options = {
    credits: {
      enabled: false
    },
```